shallow_backup.py

```diff
@@ -223,7 +223,11 @@ def backup_configs(backup_path):
 	make_dir_warn_overwrite(backup_path)
 
 	configs_dir_mapping = {"Library/Application Support/Sublime Text 2/Packages/User/": "sublime_2",
```

Comment:
I refactored this method, and you'll have to migrate this code to the `get_config_mapping()` function. Pull from master and it should be obvious where it goes.

Reply:
Yeah that'll make it much easier to add stuff in the future since it'll only have to be put in one place instead of two! 👍